pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -521,13 +547,46 @@ func (c *Cluster) getPVCInitContainerActivate(mountPath string, osdProps osdProp
 				DevicePath: fmt.Sprintf("/%s", osdProps.pvc.ClaimName),
 			},
 		},
-		VolumeMounts: []v1.VolumeMount{
+		VolumeMounts:    []v1.VolumeMount{getPvcOSDBridgeMountActivate(mountPath, osdProps.pvc.ClaimName)},
+		SecurityContext: opmon.PodSecurityContext(),
+		Resources:       osdProps.resources,
```

Comment:
Which resources are applied? We don't want to apply the same resources as the main container. The init containers should have smaller resource requirements.

Reply:
Agreed but we just don't do this AFAIK, we only make a distinction between prepare and activate, but not the init containers of the activation. Not sure if this PR is appropriate to fix this. Thoughts? We could re-use the prepare job resources perhaps?